packages/donor/src/components/AppHeader/style.module.scss

```diff
@@ -39,7 +39,17 @@
 .login {
   z-index: 20;
   margin-left: 15px;
-  margin-top: 5px;
+  display: flex;
+}
+
+.login_text {
+  color: var(--donor-link);
+  text-align: center;
+  font-size: 14px;
+  font-family: "Rubik";
```

Comment:
Do you need to set it?
It should inherit Rubik by default, no?

Reply:
You're right, I'm updating this